ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2530,23 +2530,19 @@ def download_packages(host, pkgs):
     :param host: the host where the download takes place
     :param pkgs: packages to install, provided as a list of strings
 
-    A package can't be downloaded that is already installed.
-
     Returns the temporary directory where the packages are.
     The caller is responsible for cleanup.
     """
     platform = get_platform(host)
     tmpdir = os.path.join('/tmp', str(uuid.uuid4()))
     # Only supports RHEL 8+ and Fedora for now
     if platform in ('rhel', 'fedora'):
-        install_cmd = ['/usr/bin/dnf', '-y',
-                       '--downloaddir', tmpdir,
-                       '--downloadonly',
-                       'install']
+        install_cmd = ['/usr/bin/dnf',
+                       'download']
     else:
-        raise ValueError('install_packages: unknown platform %s' % platform)
+        raise ValueError('download_packages: unknown platform %s' % platform)
     host.run_command(['mkdir', tmpdir])
-    host.run_command(install_cmd + pkgs)
+    host.run_command(install_cmd + pkgs, cwd=tmpdir)
```

Comment:
Please don't kill me, I have a few nits. Your change of "install packages" to "download_packages" inspired me.

This truly should just download packages but there are multiple references to "install" in the method. I think they should all be download, e.g. download_cmd = [..]. Also in the param pkgs, it should be download and not install. I cringe because I'm the one who mislabled things. But since you're in there...

Otherwise the content of the change LGTM.